app/adapters/publishing/redirect_adapter.rb

```diff
@@ -0,0 +1,20 @@
+require "securerandom"
+
+class Publishing::RedirectAdapter
+  def self.redirect_section(section, to:)
+    Services.publishing_api.put_content(
+      SecureRandom.uuid,
+      document_type: "redirect",
+      schema_name: "redirect",
+      publishing_app: GdsApiConstants::PublishingApi::PUBLISHING_APP,
+      base_path: "/#{section.slug}",
+      redirects: [
+        {
+          path: "/#{section.slug}",
+          type: GdsApiConstants::PublishingApi::EXACT_ROUTE_TYPE,
+          destination: to,
+        },
+      ],
+    )
+  end
+end
```

Comment:
I definitely feel our publishing apps should have models for Redirects, we seem to pass them around just as hashes a lot. Whitehall's `Unpublishing` is along the right lines but subtly different from a pure redirect in confusing ways